asb-ballerina/Module.md

```diff
@@ -18,7 +18,7 @@ applies [SAS authentication](https://docs.microsoft.com/en-us/azure/service-bus-
 supports SAS authentication as well.
 
 This module
-supports [Service Bus SDK 3.5.1 version](https://docs.microsoft.com/en-us/java/api/overview/azure/servicebus/client?view=azure-java-stable&preserve-view=true)
+supports [Service Bus SDK 7.13.1 version](https://docs.microsoft.com/en-us/java/api/overview/azure/servicebus/client?view=azure-java-stable&preserve-view=true)
 . The source code on GitHub is
 located [here](https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/servicebus/microsoft-azure-servicebus). The
```

Comment:
Let's update this pointing to the latest SDK as well
https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/servicebus/azure-messaging-servicebus